sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -578,39 +583,29 @@ case class StringTrim(children: Seq[Expression])
     val getTrimFunction = if (children.size == 1) {
       s"UTF8String ${ev.value} = ${inputs(0)}.trim();"
     } else {
-      s"UTF8String ${ev.value} = ${inputs(1)}.trim(${inputs(0)});".stripMargin
+      s"UTF8String ${ev.value} = ${inputs(1)}.trim(${inputs(0)});"
```

Comment:
I can add something like this: 
`val inputs = evals.map { eval =>
      s"${eval.isNull} ? null : ${eval.value}"
    }.reverse`
there are couple places I will add the reverse in each trim function, what do you think?